Formula/gstreamermm.rb

```diff
@@ -20,6 +19,12 @@ class Gstreamermm < Formula
 
   needs :cxx11
 
+  # Bug filed upstream with patch at https://bugzilla.gnome.org/show_bug.cgi?id=772645
```

Comment:
Necessary, I don't know about that. But it is customary. Here are some examples:
https://gist.github.com/ilovezfs/62a9eb715dc044f7202cbf86caaf713b